cfme/infrastructure/pxe.py

```diff
@@ -574,6 +580,31 @@ def delete(self, cancel=False, *ct_objs):
             else:
                 navigate_to(ct_obj, 'Details')
 
+    def copy(self, ct_obj, name=None, description=None, cancel=False):
```

Comment:
**Required**

I don't think you should take a `ct_obj` parameter here - the copy operation is against `self`. 

See other framework implementations of entity_object.copy() - they all work against `self`, with some taking `**updates` to override `self` values.